index.html

```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
           </p>
         </div>
         <div class="project-tag">
+          <ul class="project-skills flex">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+        <button class="see-project-btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>
+  <section id="about" class="about">
+    <div class="about-container flex column">
+      <div class="abt-me flex column">
+        <h2 class="header-text">About <br>Myself</h2>
+        <p class="para-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="self-container">
+          <p>Let's Connect</p>
+          <div class="list-nav">
+            <ul class="list-items skill-items">
+              <li>
+                <a href="">
+                  <img src="images/Twitter.svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <img src="images/Vector.svg" alt="my social handle images"/><img src="images/Vector (1).svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <img src="images/Medium.svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <img src="images/Vector1.svg" alt="my social handle images" />
+                </a>
+              </li>
+              <li>
+                <a href=""><img src="images/Angeist.svg" alt="my social handle images" />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <button type="button">Get my Resume</button>
+      </div>
+      <div class="language flex column">
+        <ul>
+          <li>
+            <div class="lang flex column">
+              <p class="lang-text flex">
+                <span class="txt">Language</span><span class="lang-line">></span>
+              </p>
+              <ul class="lang-div flex column">
+                <li class="lang-list flex">
+                  <img src="images/Js.svg" alt="js logo" />
+                  <p class="skill-text">JavaScript</p>
```

Comment:
You missed this `font-size` This is `15x` as per Figma design.
![image](https://user-images.githubusercontent.com/61361037/219857133-617ab936-2bec-4d24-b55f-d0b7646a6f09.png)